packages/ember-routing/lib/services/router.js

```diff
@@ -6,8 +6,33 @@ import { Service, readOnly } from 'ember-runtime';
 import { shallowEqual, resemblesURL, extractRouteArgs } from '../utils';
 
 /**
-   The Router service is the public API that provides component/view layer
-   access to the router.
+   The Router service provides components, and other services, access
+   to the router.
+   
+   It can be used to check the current route, initiate route transitions and more.
```

Comment:
`transitions, and more.`

Reply:
I personally don't like the serial comma, but it's an old debate:
https://en.wikipedia.org/wiki/Serial_comma#Arguments_for_and_against

Do you have a style guide that all documentation is supposed to follow?